definitions/variable/macro-economy/tag_econ_sectors.yaml

```diff
@@ -0,0 +1,10 @@
+- Econ sectors:
```

Comment:
Is there a place where sectors are well defined and can be used as a reference, in particular for sub-sectors. For example, referring to ISIC4 sectors?

Reply:
https://ilostat.ilo.org/resources/concepts-and-definitions/classification-economic-activities

Reply:
Agriculture would be A.
Industry B,C,D.
Services the remaining.
Correct?